develop.md

```diff
@@ -45,6 +47,9 @@ This was tricky on Ubuntu 2018:
     ALTER USER 'root'@'localhost' IDENTIFIED BY 'rootsonar';
     flush privileges;
 
+If you're using vagrant image, you may need to specify the shema by adding this parameter (update the value to the schema you're using) :
+    sonar.jdbc.schema=sonar
```

Comment:
Nitpick on markdown: please add a blank line before this code block, for a more clear visual separation from the text before it.

And, other properties settings in this document are using spaces around the `=` (because I find that easier to read).
I think it's good to be consistent, so I would add spaces here too, or if you have something against that writing style, then it would be good to change everywhere.